plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/KafkaProducerModule.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.eventlistener.kafka;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import com.google.inject.Scopes;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.trino.plugin.eventlistener.kafka.producer.KafkaProducerFactory;
+import io.trino.plugin.eventlistener.kafka.producer.PlaintextKafkaProducerFactory;
+import io.trino.plugin.eventlistener.kafka.producer.SSLKafkaProducerFactory;
+import io.trino.plugin.kafka.KafkaSecurityConfig;
+import io.trino.plugin.kafka.security.SslSecurityModule;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+
+import static io.airlift.configuration.ConditionalModule.conditionalModule;
+
+public class KafkaProducerModule
+        extends AbstractConfigurationAwareModule
+{
+    @Override
+    protected void setup(Binder binder)
+    {
+        bindSecurityModule(SecurityProtocol.SSL, new SslSecurityModule());
+        installClientModule(null, KafkaProducerModule::configureDefault);
+        installClientModule(SecurityProtocol.PLAINTEXT, KafkaProducerModule::configureDefault);
+        installClientModule(SecurityProtocol.SSL, KafkaProducerModule::configureSsl);
+    }
+
+    private void bindSecurityModule(SecurityProtocol securityProtocol, Module module)
```

Comment:
`bindSecurityModule` and `installClientModule` is almost identical. We can remove one of them if we avoid `null` protocol